### ISSUE TITLE
Accessibility:Step Input: The label “PC” is not read when step input field is accessed. 

### ISSUE THREAD

**Issue by puru-hk** [SAP/fundamental-ngx#3489]
Description: The label “PC” is not read when step input field is accessed. 

Expected: corresponding label should be read out when UI is accessed. 

Screenshot:

![image](https://user-images.githubusercontent.com/32538291/94649784-02225b00-0313-11eb-893f-3438612d7b0f.png)

**Comment by puru-hk** [comment]
@Sanjay-1312 could you please retest this issue

**Comment by Sanjay-1312** [comment]
Hello Colleagues,

The issue is fixed and hence Closing the ticket.

Thanks and Regards
Sanjay
